java/server/src/org/openqa/selenium/grid/commands/Standalone.java

```diff
@@ -119,10 +119,10 @@ public Executable configure(String... args) {
       LoggingOptions loggingOptions = new LoggingOptions(config);
       loggingOptions.configureLogging();
 
-      LOG.info("Logging configured.");
+      LOG.finest("Logging configured.");
```

Comment:
Again. This is part of the start up of the server, and the diagnostics are helpful.